web/themes/custom/itkdev/itkdev_project_theme/templates/node--project.html.twig

```diff
@@ -88,9 +88,11 @@
             <div class="mb-3">{{ projectTrack.getDescription()|raw }}</div>
             <div class="flex justify-between">
               <div class="flex justify-start">
-                <a href="{{ path('ai_screening_reports.project_track', {project_track_id: [projectTrack.id.value]}) }}">
-                  {{ include('@itkdev_project_theme/components/chart-image.svg') }}
-                </a>
+                {% if projectTrack.getType.field_report_type.value is same as('bubble_chart') %}
```

Comment:
Would it make sense to add the link (and other stuff) if the report type is set, i.e. use something along the lines of

``` twig
                {% if projectTrack.getType.field_report_type.value is not empty %}
```

?